### PR TITLE
Feature/atr 332 fbql support

### PR DESCRIPTION
**Pull request by SENya1990**
Merge after ATR-318

### CODE REVIEW

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx [hunk 1]
```diff
@@ -250,16 +250,16 @@
     <value>Enable additional diagnostics for ISV Solution Certification</value>
   </data>
   <data name="Setting_CodeAnalysis_StaticAnalysisEnabled_Description" xml:space="preserve">
-    <value>Enable Acuminator static code analysis for Visual Studio solution.</value>
+    <value>Enable Acuminator static code analysis for Visual Studio solutions.</value>
```
Comment (metallmonster) [inline src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: Were these strings reviewed?

File: src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="Acumatica.Acuminator.778b19d1-1d5e-4fcb-8edb-eb103feeab7c" Version="1.6.0" Language="en-US" Publisher="Acumatica" />
+    <Identity Id="Acumatica.Acuminator.778b19d1-1d5e-4fcb-8edb-eb103feeab7c" Version="1.7.0" Language="en-US" Publisher="Acumatica" />
```
Comment (metallmonster) [inline src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest]: Should it be here?

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx (hunk 1 above)
Comment (SENya1990) [reply src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: @metallmonster  yes by @k-s-popova. See the related jira item

File: src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest (hunk 2 above)
Comment (metallmonster) [reply src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest]: Ok

**Comment by SENya1990** [comment]
@metallmonster , @Derrekam  it's quite easy to see changes after merge of previous pull requests

**Pull request merged**